spec/e3db_spec.rb

```diff
@@ -298,9 +298,7 @@ def inc_field(rec, field)
   it 'can encrypt and decrypt locally' do
     client1 = E3DB::Client.new(client1_opts)
     type = sprintf('test-share-%s', SecureRandom.uuid)
-    # Create AK & share before writing (simulating creating AK during registration)
     client1_eak = client1.create_writer_key(type)
-    client1.share(type, client2_opts.client_id)
```

Comment:
Is there a reason we're removing this sharing from the tests?

Reply:
Wasn't necessary for the test - probably a copy/paste error originally.